### PR TITLE
front: fix simulation sheet and csv button style

### LINKED ISSUE #9891: Wrong simulation sheet and results table spacing
### What happened?

APP:
Image

Mockup :

Image

### What did you expect to happen?

Should match the mockup 

### How can we reproduce it (as minimally and precisely as possible)?

1. Open a scearnio with a train

### On which environments the bug occurs?

Recette (SNCF)

### On which browser the bug occurs?

Firefox, Google Chrome

### OSRD version (top right corner `Account` button > `Informations`)

deac2f4

### PR DESCRIPTION
**Pull request by Akctarus**
closes #9891 

https://www.sketch.com/s/2fb064cd-c483-46ac-bd30-72705fd63e75/p/9559F6FC-69EE-4DB8-BE1A-77F60F10C97F/canvas#Inspect

### CODE REVIEW

**Review by theocrsb — APPROVED** [review]
LGTM and tested :)

**Review by Uriel-Sautron — COMMENTED** [review]
Buttons are 32 px on sketch, but 40px on app

File: front/src/modules/simulationResult/SimulationResultExport/SimulationResultsExport.tsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ const SimulationResultExport = ({
           exportTrainCSV(simulatedTrain, operationalPoints, pathElectrifications, train)
         }
         variant="Quiet"
-        label=".CSV"
+        label=".csv"
```
Comment (Uriel-Sautron) [inline front/src/modules/simulationResult/SimulationResultExport/SimulationResultsExport.tsx:89]:
```suggestion
        label=".csv"
        size="medium"
```

Comment (Uriel-Sautron) [reply front/src/modules/simulationResult/SimulationResultExport/SimulationResultsExport.tsx:89]: Same for simulation sheet button

Comment (Akctarus) [reply front/src/modules/simulationResult/SimulationResultExport/SimulationResultsExport.tsx:89]: done ! ty for the comment

**Review by Uriel-Sautron — APPROVED** [review]
LGTM !!!

**Review by Math-R — APPROVED** [review]
Lgtm thanks !

**Pull request merged**